src/jit/compiler.h

```diff
@@ -7453,7 +7468,8 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         bool compUseFCOMI;
         bool compUseCMOV;
 #ifdef _TARGET_XARCH_
-        bool compCanUseSSE2; // Allow CodeGen to use "movq XMM" instructions
+        bool compCanUseSSE2;   // Allow CodeGen to use "movq XMM" instructions
+        bool compCanUseSSE3_4; // Allow CodeGen to use SSE3, SSSE3, SSE4.1 and SSE4.2 instructions
 
```

Comment:
(Similarly, it seems like `compCanUseAVX` should be `compCanUseAVX2`.)

---
In reply to: [89865129](https://github.com/dotnet/coreclr/pull/8291#discussion_r89865129) [](ancestors = 89865129)